### ISSUE TITLE
Bug: table alias missing

### ISSUE THREAD

**Issue by ReinierKoops** [macbre/sql-metadata#184]
Hi,

I think I have found a bug regarding table aliases. I have the following query:

`query = "select distinct e1.eid as eid from uw_emp e1, uw_emp e2, uw_payroll p where e1.eid = e2.eid and e1.eid = p.eid"`
`print(query.columns, "\n", query.columns_aliases_names, "\n", query.tables, "\n", query.tables_aliases)`

which outputs:
`['e1.eid', 'e2.eid', 'uw_payroll.eid'] `
` ['eid'] `
` ['uw_emp', 'uw_payroll'] `
` {'p': 'uw_payroll'}`

I am missing e1 and e2 for aliases.

**Comment by ReinierKoops** [comment]
I think it has something to do with the fact that I am referencing the same table twice in the query.

**Comment by collerek** [comment]
There `sqlparse` treat `e1` and `e2` as floats, I expect it has to do something with exponentials in `sqlparse`?

If you change aliases to something else it should work, i.e. `em` instead of `e`:
```python
query = """
    select distinct em1.eid as eid from uw_emp em1, uw_emp em2, uw_payroll p 
    where em1.eid = em2.eid and em1.eid = p.eid
    """
```

If you want to keep the old aliases you have to quote them, i.e.:
```python
query = """
    select distinct e1.eid as eid from uw_emp "e1", uw_emp "e2", uw_payroll p 
    where e1.eid = e2.eid and e1.eid = p.eid
    """
parser = Parser(query)
assert parser.tables == ["uw_emp", "uw_payroll"]
assert parser.tables_aliases == {
    'p': 'uw_payroll',
    'e1': 'uw_emp',
    'e2': 'uw_emp'
}
```

**Comment by collerek** [comment]
It's even reported https://github.com/andialbrecht/sqlparse/issues/399 since... 2018 😅

**Comment by collerek** [comment]
It was fixed in `sqlparse` but was not yet released (0.4.2), so you need to install it from github before the next release.

**Comment by ReinierKoops** [comment]
Thank you, this is great! I see its merged to their master branch. I will install that version.
